backend/.golangci.yml

```diff
@@ -3,21 +3,21 @@ linters:
   disable-all: true
   enable:
   - deadcode
-  - gci
   - errcheck
+  - exportloopref
```

Comment:
may want to use https://github.com/kyoh86/looppointer instead, it's a stricter version https://github.com/kyoh86/looppointer#sensing-policy

Reply:
This is not a supported linter by upstream golangci-lint, we could use it but would require more work to compile if I'm understanding the private plugin system correctly.

Reply:
ah carry on then, we can take our chances with the less strict version